index.html

```diff
@@ -41,442 +19,960 @@
-              <strong class="step-bold"> Step 5 -</strong> Your subscribers will
-              receive a notification each time you post a new product. They can
-              instantly buy and post a product review, like or share your
-              product post.
-            </p>
-            <p class="how_it_works">
-              <strong class="step-bold"> Step 6 -</strong> For every purchase made
-              allows the buyer to write a product review for the item being
-              bought.
-              <span class="more" id="more-six"
-                >Alternatively, the buyer can as well navigate to visit your
-                store offline to make patronage. Sellers can reward their
-                subscribers as affiliates for each patronage made through the
-                subscriber's page whenever they share the seller's product.
-              </span>
-              <span class="more-btn" id="more-btn-six">read more</span>
-            </p>
+          <a href="#contact" class="btn-get-started scrollto">Get Started</a>
+          <a
+            href="./assets/files/turung_whitepaper.pdf"
+            class="btn-projects scrollto"
+            >Our Projects</a
+          >
         </div>
-      </section>
+      </div>
 
-      <section class="benefits-con">
-        <div class="benefits-desc">
-          <h1>BENEFITS</h1>
-          <p>
-            Living in a consumer-based society, integrating traditional business
-            with a physical presence on Turung smartstore gives them the
-            superpowers to effectively operate as mini banks with the following
-            benefits;-
-          </p>
-        </div>
-        <div class="benefits">
-          <div class="benefit benefit-one">
-            <div class="benefit-img">
-              <img src="./images/clock-with-lines.jpg" alt="clock with lines" />
+      <div id="intro-carousel" class="owl-carousel">
+        <div
+          class="item"
+          style="
+            background-image: url('assets/images/intro-carousel/pattern4.jpg');
+          "
+        ></div>
+        <div
+          class="item"
+          style="
+            background-image: url('assets/images/intro-carousel/pattern1.jpg');
+          "
+        ></div>
+        <div
+          class="item"
+          style="
+            background-image: url('assets/images/intro-carousel/pattern2.jpg');
+          "
+        ></div>
+        <div
+          class="item"
+          style="
+            background-image: url('assets/images/intro-carousel/pattern3.jpg');
+          "
+        ></div>
+      </div>
+    </section>
+    <!-- #intro -->
+
+    <main id="main">
+      <!--==========================
+      About Section
+    ============================-->
+      <section class="about wow fadeInUp bg-black">
+        <div class="container">
+          <div class="row">
+            <div class="col-lg-4 about-img">
+              <img src="assets//images/turung-logo.png" alt="" />
```

Comment:
Hi Victoria, thanks for feedback. its has been solved out

Reply:
![image](https://user-images.githubusercontent.com/60205816/179203013-451cdd14-bdaa-42a8-a0e2-819ebead03ed.png)

Since we are still talking about images :smile: , I think the icons are supposed to be centred, or what do you think?
If that's how the design came then don't worry. It's a suggestion from my end

Reply:
ok. noted

Reply:
> Hi Victoria, thanks for feedback. its has been solved out

Nice work :clap: